ecosystem/resources/members/vqls-prototype.toml

```diff
@@ -1,6 +1,6 @@
 name = "Variational Quantum Linear Solver Prototype"
 url = "https://github.com/QuantumApplicationLab/vqls-prototype"
-description = "The Variational Quantum Linear Solver (VQLS) uses an optimization approach to solve linear systems of equations. The vqls-prototype allows to easily setup and deploy a VQLS instance on different backends through the use of qiskit primitives and the runtime library."
+description = "VQLS uses an optimization approach to solve linear systems of equations. Set up and deploy a VQLS instance on different backends."
```

Comment:
```suggestion
description = "VQLS uses an optimization approach to solve systems of linear equations. Set up and deploy VQLS instances on different backends."
```

Reply:
"linear systems of equations" isn't a thing; right? I know a "system of linear equations" is a thing.

Reply:
I think you're right